### ISSUE TITLE
Sorry to bother you again

### ISSUE THREAD

**Issue by cervineee** [LanglandsLin/ActCLR#6]
Sorry to bother you again, my pretext is always around 77%, the ntu60_xview_joint.pt in the released_model used, the data is action_dataset/ntu60_frame50/xview/train_position.npy,the environment is installed according to the required content， please where do I need to change to improve

**Comment by cervineee** [comment]
I see that the efficiency in your log.txt can reach 79%

**Comment by LanglandsLin** [comment]
This is correct  because the pre-trained model reaches 79% of the optimum at 300 epoch and then drops to 77%. The weights we put out are for 350 epoch, at which point the knn accuracy is 77%. But this sometimes improves in downstream linear evaluation, and we think this may be caused by mitigating overfitting.

**Comment by cervineee** [comment]
Your pre-training results are:
300-10: cur-79.34%, best-79.43%
250-40: cur-76.48%, best-77.04%
300-20: cur-78.88%, best-79.07%
300-10: cur-79.34%, best-79.43%
300-40: cur-78.19%, best-78.36%
300-20: cur-78.88%, best-79.07%
300-40: cur-78.19%, best-78.36%
 I train for: 
250-10: cur-76.66%, best-76.66%
250-20: cur-76.43%, best-76.43%
250-40: cur-75.44%, best-75.44%
300-10: cur-77.86%, best-77.98%
300-20: cur-77.73%, best-77.73%
300-40: cur-76.88%, best-77.02%
 Do I have any hope of running your results?

**Comment by LanglandsLin** [comment]
Are you using linear_evaluation.py in skeleton_recognition? The error could be because our pre-trained st-gcn outputs multiple features, but the downstream recognition only needs global features, so we need to use the  st-gcn in skeleton_recognition.

**Comment by LanglandsLin** [comment]
Regarding the KNN performance degradation, I'll give the AimCLR weights I used for initialisation for reference.

**Comment by cervineee** [comment]
Hello, I copied the linear_eval in the skeleton folder to the config of the whole folder, and the other required py files were also copied out, do you have to use linear_evaluation.py in skeleton

**Comment by cervineee** [comment]
The code of AIMCL running in Skeleton, ST-GCN is also the original GCN, so how does the evaluation of ACTCL run?

**Comment by cervineee** [comment]
Hello, I'm in linear_eval.._joint.yaml file: Set 
model: net.actclr.actclr
model_args: base_encoder: net.st_gcn. model
where the st_gcn is the original gcn, which runs your results epoch100_acc87.14.pt in the work_dir
Is this the result of the NTU 60 xview (%) of ActCLR-joint?

**Comment by LanglandsLin** [comment]
yes
